app/api/gift-exchanges/[id]/send-email/route.ts

```
@@ -0,0 +1,53 @@
+ * @returns JSON response indicating success or failures.
+ */
+export async function POST(req: NextRequest): Promise<NextResponse> {
+  const resend = new Resend(process.env.RESEND_API_KEY);
+
+  try {
+    const body = await req.json();
+    const { members, suggestions } = body;
+
+    // Send emails to all members
+    for (const member of members) {
+      let recommendedGifts = [];
```

ESLint

[error] 'recommendedGifts' is never reassigned. Use 'const' instead.

(prefer-const)